free-programming-books-ua.md

```diff
@@ -16,6 +21,11 @@
 * [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
 
 
+### JavaScript
+
+* [Розуміння ECMAScript 6](http://understandinges6.denysdovhan.com/) — Денис Довгань (LambdaBooks)
```

Comment:
Travis CI linter wants you to "Remove trailing slash (http://understandinges6.denysdovhan.com)"